app/notifications/rest.py

```diff
@@ -83,9 +85,13 @@ def send_notification(notification_type):
         msg = msg + ", please use the latest version of the client" if notification_type == LETTER_TYPE else msg
         raise InvalidRequest(msg, 400)
 
-    notification_form, errors = (
-        sms_template_notification_schema if notification_type == SMS_TYPE else email_notification_schema
-    ).load(request.get_json())
+    schema = sms_template_notification_schema if notification_type == SMS_TYPE else email_notification_schema
+
+    errors = {}
+    try:
+        notification_form = schema.load(request.get_json())
+    except ValidationError as e:
+        errors = e.messages
 
     if errors:
         raise InvalidRequest(errors, status_code=400)
```

Comment:
@leohemsted There actually already is a marshmallow error handler in app/errors.py: https://github.com/alphagov/notifications-api/blob/00a04ebf54c97fc695f013de0a497e5490ddb558/app/errors.py#L59-L62

I could just stop catching the ValidationErrors and let that take care of things. I was trying to keep things as close to how they currently are as possible for now to try and avoid anything breaking, so had kept the `InvalidRequest` error. But I could either change this now, or make the change in a second PR once this is deployed if that's a little safer.

> and any validation error would return a 400 rather than a 500.

I'm not sure see how this changes the status code - both look like they would return a `400` to me?

Reply:
This is now updated to use the marshmallow error handler